comms/discovery/pubkeystore/recover.go

```diff
@@ -140,6 +139,61 @@ func RecoverUserPublicKeyBase64(ctx context.Context, userId int) (string, error)
 
 	}
 
+	rows.Close()
```

Comment:
should we remove the "try POA" steps above?

Reply:
Yeah I guess POA is a lost cause at this point.  Doing that will let us consolidate the two steps.

Reply:
Going to merge this now and come back to do that cleanup